roles/vault/defaults/main.yml

```diff
@@ -10,7 +10,7 @@ vault_docker_tag: "latest"
 
 vault_cluster_name: ""
 # Allow vault_vip_url for backwards compatibility.
-vault_vip_address: "{{ vault_vip_url | default('') }}"
+vault_api_addr: "{{ vault_vip_url | default('') }}"
```

Comment:
Do we need to support the old name?

```
vault_api_addr: "{{ ('https://' ~ vault_vip_address ~ ':8200') if vault_vip_address else '' }}"
vault_vip_address: "{{ vault_vip_url | default('') }}"
```

(I gave this pattern a very quick test)